security/SqlDbInsecureApp/Pages/Contact.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
+using Dapper;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SqlDbInsecureApp.Pages
+{
+    public class ContactModel : PageModel
+    {
+        private readonly IConfiguration config;
+
+        public string Message { get; set; }
+        [BindProperty(SupportsGet = true)]
+        public string GenerateLoad { get; set; }
+
+        public ContactModel(IConfiguration config)
+        {
+            this.config = config;
+        }
+
+        public async Task OnGet()
+        {
+            Message = "Your contact page.";
+            if (string.IsNullOrWhiteSpace(GenerateLoad)) return;
+            
+            // run a query to try to trigger an automatic tuning recommendation (didn't work)

```

Comment:
Did you mean to keep this comment in if these samples will be shared widely?

Reply:
I think it's fine to be there for now. Ideally we rip out this page eventually since it doesn't do anything useful.